### PR TITLE
feat(web): add dashboard

### PR DESCRIPTION
**Pull request by balanceee**

### CODE REVIEW

File: protobuf/proto/crux.proto [hunk 1]
```diff
@@ -1083,3 +1086,38 @@ message CreateProductFromTemplateRequest {
   string description = 101;
   ProductType type = 102;
 }
+
+//Dashboard
+
```
Comment (peterambrus3) [inline protobuf/proto/crux.proto:1091]: remove newline

File: protobuf/proto/crux.proto [hunk 2]
```diff
@@ -1083,3 +1086,38 @@ message CreateProductFromTemplateRequest {
...
+  uint32 auditLogEntries = 101;
+  uint32 products = 103;
+  uint32 versions = 104;
+  uint32 deployments = 105;
+  uint32 failedDeployments = 106;
+
+  repeated DashboardActiveNodes nodes = 1000;
+  repeated DashboardDeployment latestDeployments = 1001;
+  repeated AuditLogResponse auditLog = 1002;
+}
```
Comment (peterambrus3) [inline protobuf/proto/crux.proto:1123]: no newline

File: web/crux-ui/e2e/utils/global-setup.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
+import { ROUTE_DASHBOARD } from '@app/routes'
 /* eslint-disable import/no-extraneous-dependencies */
-import { ROUTE_LOGIN, ROUTE_PRODUCTS, ROUTE_TEAMS_CREATE, verificationUrl } from '@app/routes'
+import { ROUTE_LOGIN, ROUTE_TEAMS_CREATE, verificationUrl } from '@app/routes'
```
Comment (polaroi8d) [inline web/crux-ui/e2e/utils/global-setup.ts:3]: What's the difference between lines 1 and 3?

File: web/crux/src/app/dashboard/dashboard.service.ts [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+          product: {
+            teamId: team.teamId,
+          },
+        },
+        status: DeploymentStatusEnum.failed,
+      },
+    })
+
+    const activeNodes = await this.prisma.node.findMany({
+      where: {
```
Comment (polaroi8d) [inline web/crux/src/app/dashboard/dashboard.service.ts:68]: It can't be replaced with `teamFilter`?

File: protobuf/proto/crux.proto [hunk 5]
```diff
@@ -1083,3 +1086,37 @@ message CreateProductFromTemplateRequest {
...
+  string changelog = 103;
+  optional google.protobuf.Timestamp deployedAt = 104;
+  string productId = 105;
+  string versionId = 106;
+}
+
+message DashboardResponse {
+  uint32 users = 100;
+  uint32 auditLogEntries = 101;
+  uint32 products = 103;
```
Comment (robot9706) [inline protobuf/proto/crux.proto:1114]: 102 missing?

**Review by robot9706 — CHANGES_REQUESTED** [review]
Minor comment, also testing in progress. 🚀

**Review by robot9706 — APPROVED** [review]
LGTM, tested 🚀

**Review by peterambrus3 — APPROVED** [review]

**Pull request merged**